assignments/arrays.js

```diff
@@ -63,17 +63,36 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+
+// console.log(`Car 33 is a` *car year goes here*' *car make goes here* *car model goes here*` );
+
+
+
+// console.log("Car", inventory[32].id, "Is a", inventory[32].car_year, inventory[32].car_make, inventory[32].car_model );
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+    let lastCar = 50 -1;
```

Comment:
Instead of hard coding a more flexible solution code have been: inventory.length -1, this way you'll always access the last item even if more items are added or taken out.